### PR TITLE
add trail blog post and data page update

### PR DESCRIPTION
**Pull request by gregbunce**
also, add agol link back to the roads data page.

### CODE REVIEW

**Comment by gregbunce** [comment]
Note, the downloadable data on the trails data page currently include these updates and changes.

**Comment by gregbunce** [comment]
i'm out of the office next week, so feel free to make any necessary changes before it's merged.  thanks!

File: data/recreation/trails/index.html [hunk 1]
```diff
@@ -38,28 +38,43 @@
...
+    <p>Currently, the content primary depicts features used by hikers/pedestrians and bikers. It began with map data purchased from a private entity (Orbital View, Inc), for which full rights were purchased by the GOED Office of Outdoor Recreation in July 2014. Since the original purchase, subsequent substantive updates have included: 2014 Revisions from National Park service; 2015 additions from Emery County; June 2016 additions from Cache County US Forest Service Geodata Hub (all forests except Sawtooth in NW Box Elder County), and Mountainland Association of Governments (Utah and Wasatch counties); 2018/2019 additions as part of the <a href="{% link data/transportation/street-network-analysis/index.html %}#MultimodalNetwork">multimodal project</a>; and currently ongoing updates by WFRC transportation planners.</p>
+    <p>Presently, the planned approach for on street bicycle lanes and routes is to incorporate them in the the <a href="{% link data/transportation/roads-system/index.html %}">statewide roads</a> GIS layer using left and right side attributes. On street bike infrastructure is not expected to be represented in this map layer.</p>
```
Comment (steveoh) [inline data/recreation/trails/index.html:47]:
```suggestion
    <p>Presently, the planned approach for on street bicycle lanes and routes is to incorporate them in the the <a href="{% link data/transportation/roads-system/index.html %}">statewide roads</a> GIS layer using left and right side attributes. On street bike infrastructure is not expected to be represented in this map layer.</p>
```

File: data/recreation/trails/index.html [hunk 2]
```diff
@@ -38,28 +38,43 @@
...
-      <dt>DataSource</dt>
-      <dd><a href="https://docs.google.com/spreadsheets/d/1OPGKmRrLYUBr9qO9EtbX6AwcRla82xMvc-BDr26ia9k/edit?usp=sharing">Full Data Model</a>
+      <dt>Class</dt>
+      <dd>For a general query of facility types, use the Class field:
+        <ul class="dotless">
+          <li> Path - Paved, off-street travelway designed for non-motorized users, generally wider and with greater separation from roadway than a standard sidewalk
+          <li> Trail - Natural surface travelway (dirt, gravel, slickrock, snow, etc)
+          <li> Link - Network route connection through a parking lot, alleyway, etc. where a road segment does not exist but can be traversed via walking or bicycling
+          <li> Steps - Developed (generally concrete) steps or stairs, usually occurring along a paved route
+          <li> Grade Separated Crossing - Paved crossing feature above or below roadway, train track, etc. for walking and bicycling use only
```
Comment (steveoh) [inline data/recreation/trails/index.html:72]:
```suggestion
          <li>Path - Paved, off-street travelway designed for non-motorized users, generally wider and with greater separation from roadway than a standard sidewalk
          <li>Trail - Natural surface travelway (dirt, gravel, slickrock, snow, etc)
          <li>Link - Network route connection through a parking lot, alleyway, etc. where a road segment does not exist but can be traversed via walking or bicycling
          <li>Steps - Developed (generally concrete) steps or stairs, usually occurring along a paved route
          <li>Grade Separated Crossing - Paved crossing feature above or below roadway, train track, etc. for walking and bicycling use only
```

**Pull request merged**